client/webserver/site/src/js/app.js

```diff
@@ -658,6 +658,19 @@ export default class Application {
     return o
   }
 
+  haveAssetOrders (assetID) {
+    for (const xc of Object.values(this.user.exchanges)) {
+      for (const market of Object.values(xc.markets)) {
+        if (!market.orders) continue
+        for (const ord of market.orders) {
+          if ((ord.baseID === assetID || ord.quoteID === assetID) &&
+            (ord.status < StatusExecuted || hasLiveMatches(ord))) return true
+        }
+      }
+    }
+    return false
+  }
+
```

Comment:
We no longer need `haveAssetOrders` since `*Core` does it now.

Reply:
Will removed.  Although I thought it was a good complement to the backend check to avoid a round trip api request looking for the error code, and would also be nice when a Force Rescan prompt is added.  It's no matter though, might as well have a single source of logic on this.